specification/app/resource-manager/Microsoft.App/stable/2025-01-01/ContainerApps.json

```diff
@@ -853,6 +857,22 @@
         }
       }
     },
+    "Runtime": {
+      "description": "Container App Runtime configuration.",
+      "type": "object",
+      "properties": {
+        "java": {
+          "description": "Java app configuration",
+          "type": "object",
+          "properties": {
+            "enableMetrics": {
+              "description": "Enable jmx core metrics for the java app",
+              "type": "boolean"
```

Comment:
Consider using enum instead of booleans : 

[https://github.com/Azure/azure-openapi-validator/blob/main/docs/enum-instead-of-boolean.md](https://github.com/Azure/azure-openapi-validator/blob/main/docs/enum-instead-of-boolean.md)


for example : MetricsState : Enabled\Disbled\Critical\Verbose etc

Reply:
Thanks @raosuhas, I understand the benefit to use enum here.
Here we are publishing new stable version and pick some features from older preview versions.
I think we should keep this type to the preview versions, so we won't make any break changes.

Reply:
ok please keep these in mind at the beginning even for the preview versions. Approving this one for now